pinot-core/src/main/java/org/apache/pinot/core/util/CrcUtils.java

```diff
@@ -81,8 +88,9 @@ public long computeCrc()
         }
       }
     }
-
-    return checksum.getValue();
+    long crc = checksum.getValue();
+    LOGGER.info("Computed crc = {}, based on files {}", crc, Arrays.toString(_files.toArray()));
```

Comment:
`LOGGER.info("Computed crc = {}, based on files {}", crc, _files);`
Same for MD5